README.md

```diff
@@ -11,11 +11,31 @@ First install XCHammer
 make install
 ```
 
-Generate using a [XCHammerConfig](https://github.com/pinterest/xchammer/blob/master/Sources/XCHammer/XCHammerConfig.swift). 
+Generate using a [XCHammerConfig](https://github.com/pinterest/xchammer/blob/master/Sources/XCHammer/XCHammerConfig.swift).
 ```
 xchammer generate <configPath>
 ```
 
+## Integration with your Project (Bazel)
+
+Add the following to your `WORKSPACE` file
+```
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "xchammer",
+    remote  = "https://github.com/pinterest/xchammer"
+)
+
+load("@xchammer:third_party:repositories.bzl", "xchammer_dependencies")
+
+xchammer_dependencies()
+
+```
+
+Then you can run `bazel build @xchammer//:xchammer` to compile from source.
```

Comment:
I think compiling with release is something that we could certainly mention but isn't necessarily something we should have to call out

Reply:
I agree and entire Bazel example command is something we shouldn’t have to call out as well.
If we keep it, it might save people trouble by adding the relevant chars to the example: “-c opt”. People trying this project might not be familiar with Bazel when they try to setup XCHammer for the first time